src/OmniSharp.Abstractions/Models/v1/FileMemberElement.cs

```diff
@@ -10,6 +10,8 @@ public class FileMemberElement : IComparable<FileMemberElement>
         public QuickFix Location { get; set; }
 
         public string Kind { get; set; }
+        
+        public ICollection<string> Features { get; } = new List<string>();
```

Comment:
So this collection just contains metadata for the particular element? (in this case it's a test method).

It kind of feels like we should expand this a little bit.

What if we added a new class... (the name doesn't matter to me)

```
public enum FeatureType {
    TestMethod
}

public class FeatureItem
{
    public FeatureType Type { get; set; }
    public string Name { get; set; }
}
```

Here's my reasoning.  We could expand this out to include additional items such as has the test passed already, the last reference count (code lens).  This way we leave this object open for extension, should we choose to do so.